.editorconfig

```diff
@@ -0,0 +1,28 @@
+root = true
+
+[*]
+end_of_line = LF
+charset = utf-8
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.html]
+indent_size = 2
```

Comment:
HTML gets super nested. It kinda makes sense bring it in a bit. I'm not that fussed though